### DISCUSSION TITLE
Support optional branches without warnings

### DISCUSSION THREAD

**Discussion by Donnerbart** [renovatebot/renovate#31156] [Suggest an Idea]
### Tell us more.

We use Renovate in a big platform project that consists of many projects. To quickly test renovate configurations, we have a playground branch configured in all repositories, so we don't need to add/remove this just to test something. For example:
```
    baseBranches: [
        "main",
        "renovate-playground",
    ],
```
Most of the time this leads to a `WARN: Base branch does not exist - skipping` in all dependency dashboards.

I think it would be great to have a configuration option for such optional branches. There might be other use-cases beside a playground, e.g. release branches.

Alternatively add the missing branch name in the warning, so users can quickly see what's wrong with the configuration (or in case of an optional branch, quickly see that nothing is actually wrong).

**Comment by rarkins** [comment]
You can use `logLevelRemap` to lower the level of this message

**Reply by Donnerbart** [reply]
That's good to know, but could also hide a misconfiguration for branches that should not be optional. But I will propose this as an alternative to my colleagues.

If this idea is not accepted, I would still much appreciate the branch name in the log message, e.g. `WARN: Base branch 'renovate-playground' does not exist - skipping`.

**Reply by rarkins** [reply]
We do not put variable strings in warning or error messages as otherwise you can't collect statistics about how often they're being raised
